app/javascript/admin/controllers/data_update_script_controller.js

```diff
@@ -0,0 +1,114 @@
+import { Controller } from 'stimulus';
+
+export default class DataUpdateScriptController extends Controller {
+
+  forceRun(event) {
```

Comment:
Would that have worked by giving the same target to every script and adding a `data-value` with the `id` to fetch inside Stimulus? 

Stimulus, since 2.0, supports [multiple HTML elements with the same target](https://stimulus.hotwire.dev/reference/targets):

> Plural | this.[name]Targets | An array of all matching targets in scope

Anyway, probably not worth revisiting for now

Reply:
Yes possibly, I must have been looking at slightly older docs. I'm going to add a note for possibly my cooldown to refactor out this code and try it this way.